docs-source/content/samples/simple/azure-kubernetes-service/troubleshooting.md

```diff
@@ -0,0 +1,334 @@
+   - [System pods are pending](#the-aks-cluster-system-pods-are-pending)
+   - [WebLogic Kubernetes Operator ErrImagePull](#weblogic-kubernetes-operator-errimagepull)
+- [WSL2 bad timestamp](#wsl2-bad-timestamp)
+- [Cannot attach ACR due to not being Owner of subscription](#cannot-attach-acr-due-to-not-being-owner-of-subscription)
+- [Virtual Machine size is not supported](#virtual-machine-size-is-not-supported)
+
+#### Get pod error details
+
+You may get the following message while creating the WebLogic domain: "the job status is not Completed!"
+
+```text
+status on iteration 20 of 20
+pod domain1-create-weblogic-sample-domain-job-nj7wl status is Init:0/1
+The create domain job is not showing status completed after waiting 300 seconds.
+Check the log output for errors.
+Error from server (BadRequest): container "create-weblogic-sample-domain-job" in pod "domain1-create-weblogic-sample-domain-job-nj7wl" is waiting to start: PodInitializing
+[ERROR] Exiting due to failure - the job status is not Completed!
+```
+
+You can get further error details by running `kubectl describe pod`, as shown here:
+
+```bash
+$ kubectl describe pod <your-pod-name>
+```
+
+This is an output example:
+
+```bash
+$ kubectl describe pod domain1-create-weblogic-sample-domain-job-nj7wl
+Events:
+Type     Reason       Age                  From                                        Message
+----     ------       ----                 ----                                        -------
+Normal   Scheduled    4m2s                 default-scheduler                           Successfully assigned default/domain1-create-weblogic-sample-domain-job-qqv6k to aks-nodepool1-58449474-vmss000001
+Warning  FailedMount  119s                 kubelet, aks-nodepool1-58449474-vmss000001  Unable to mount volumes for pod "domain1-create-weblogic-sample-domain-job-qqv6k_default(15706980-73cb-11ea-b804-b2c91b494b00)": timeout expired waiting for volumes to attach or mount for pod "default"/"domain1-create-weblogic-sample-domain-job-qqv6k". list of unmounted volumes=[weblogic-sample-domain-storage-volume]. list of unattached volumes=[create-weblogic-sample-domain-job-cm-volume weblogic-sample-domain-storage-volume weblogic-credentials-volume default-token-zr7bq]
+Warning  FailedMount  114s (x9 over 4m2s)  kubelet, aks-nodepool1-58449474-vmss000001  MountVolume.SetUp failed for volume "wls-azurefile" : Couldn't get secret default/azure-secrea
+```
+
+#### Fail to access Administration Console
+
+Here are some common reasons for this failure, along with some tips to help you investigate.
+
+* **Create WebLogic domain job fails**
+
+Check the deploy log and find the failure details with `kubectl describe pod podname`.
+Please go to 1. Getting pod error details.
+
+* **Process of starting the servers is still running**
+
+Check with `kubectl get svc` and if domainUID-admin-server, domainUID-managed-server1 and domainUID-managed-server2 are not listed,
```

Comment:
Indent
domainUID-admin-server, domainUID-managed-server1 and domainUID-managed-server2 -> `domainUID-admin-server`, `domainUID-managed-server1`, and `domainUID-managed-server2` (code font)